Curve-finance-suite/Address-Provider-Ownership-Transfer/README.md

```diff
@@ -0,0 +1,26 @@
+# Transfer Ownership Agent for Registry Contract
+
+## Description
+
+This agent detects `CommitNewAdmin` event emission in the address provider contract. 
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- CURVE-4
```

Comment:
Looks like you are using `CURVE-4` here and `CURVE-3` when creating the handler in `agent.ts`